server/app.js

```diff
@@ -5,6 +5,12 @@ var logger = require("morgan");
 var cors = require("cors");
 var dao = require("./dao/dao");
 
+// ----------
+// /timeManager
+// ----------
+const { TimeManager } = require("./models/timeManager");
+var timeManagerObj = new TimeManager(); 
```

Comment:
Call this WeekPhaseService. I don't know why a service is in the models folder. Move all the code you wrote in the service folder, under a subfolder named WeekPhaseService. The main file should have the same name as the subfolder.

Also, this service has to work as the dao. We need it to be single instance for testing. Copy the dao initialization and get rid of the class.